modules/social_features/social_event/social_event.install

```diff
@@ -1621,3 +1621,36 @@ function social_event_update_8904() {
 function social_event_update_8905() {
   \Drupal::service('module_installer')->install(['social_event_addtocal']);
 }
+
+/**
+ * Updated field_enroll_method text and labels.
+ */
+function social_event_update_8906() {
+  /** @var \Drupal\update_helper\Updater $updateHelper */
+  $updateHelper = \Drupal::service('update_helper.updater');
+
+  // Execute configuration update definitions with logging of success.
+  $updateHelper->executeUpdate('social_event', 'social_event_update_8906');
+
+  // Output logged messages to related channel of update execution.
+  return $updateHelper->logger()->output();
+}
+
+/**
+ * Update the enrollment method text of an event on event add form.
+ *
+ * Load in a config file from an specific update hook that will never change.
```

Comment:
This only goes for static files, in this case you generated a nice update helper one. :D